ch12/linear_multiple.md

```diff
@@ -1 +1 @@
-!INCLUDE "../notebooks-html/linear_multiple.html"
+!INCLUDE "../notebooks-html\linear_multiple.html"
```

Comment:
Hmm. It looks like the reverse backslashes `\` should be regular slashes `/`. This looks like a issue with running the conversion scripts on Windows. For now, can you replace the `\` with `/` for this `.md` file?